learn-CSS-by-Building-a-blog/blog39.html

```diff
@@ -0,0 +1,4 @@
+<!-- 
+  We are going to use the same structure as the previous HTML page.
```

Comment:
In the final version of the project we won't be using the individual file names, just `index.html`, and perhaps `article1.html` and `article2.html`. If this is for `article1.html`, this step could read:

```
<!--
  We are going to use the same structure as the previous HTML page.
  Copy all the content inside the `index.html` file and paste it here.
-->
```

Reply:
Le all the HTML files are going to be together. Wouldn’t this make it confusing to distinguish between the actual index.html and the “blog.html”? Should we have a folder for all index.html, “blog.html” (index.html), and so on?

Reply:
@RafaelDavisH, there's still some things we need to figure out about how the new platform will work, particularly with multiple files. The idea now is that we'll have a way to iterate through the different versions of each file in the background (`index01.html`, `index02.html`), but the learners will only see `index.html`.

I think the file structure is fine for now, but we can always change it in the future. In the test descriptions you're writing now we can just refer to each file by the name learners will see like `index.html`.

Reply:
got it 👍